.env.docker

```diff
@@ -0,0 +1,25 @@
+# This file can be used to declare environment variables.
+# N.B. the image will use the contents of this file from when it was built.
+#
+# After building the docker image, one way to use this file is by running:
+#   docker run -e NODE_ENV=docker --rm -it -p 127.0.0.1:8080:80 substrate-api-sidecar:$VERSION
+#
+# If you would like to build an image with an .env file of a different name
+# simply add `!.env.myfile` to the .dockerignore.
+
+# Prior to building the Docker image you can set the ws url here. If it is not
+# set it will fallback to the default value specified in `specs.yaml` or a
+# value specified when the image is run.
+#
+# SAS_SUBSTRATE_WS_URL=wss://{my-node-ws-url}
```

Comment:
I am leaving this for now. I prefer to keep default values in the `specs.yaml` so people can have the first start up of Sidecar be easier, although I am open to change if there is a compelling UX reason.

Reply:
I think the here is that you assume that people will have a node running on localhost:9944 which will likely not be the case. So it will fail, and people will not know where it comes from.

Reply:
@joepetrowski and @danforbes - any opinion on if the default node ws url should be kept as local node or should be switched. to be empty?

Reply:
I do think most people will run sidecar on the machine where they host a node. I think leaving this as local and an error message saying that it can't find a node running, go to ... and do ... to fix it.

Reply:
I don't think it's safe to assume people will run Sidecar on the same machine as the node. On the contrary, I would assume that most operators would want to make sure that the load on Sidecar did not impact the performance of the node.